docs/usage/manifest.md

```diff
@@ -0,0 +1,84 @@
+# Manifest files
+
+## Project.swift
+
+Projects are defined in `Project.swift` which we refer to as manifest files. Manifests need to import the framework `ProjectDescription` which contains all the classes and enums that are available for you to describe your projects. The snippet below shows an example project manifest:
+
+```swift
+let project = Project(name: "MyProject",
+                      targets: [
+                          Target(name: "App",
+                                 platform: .iOS,
+                                 product: .app,
+                                 bundleId: "io.tuist.App",
+                                 infoPlist: "Config/App-Info.plist",
+                                 sources: ["Sources/**"],
+                                 resources: ["Resources/**"],
+                                 dependencies: [
+                                     .project(target: "Framework1", path: "../Framework1"),
+                                     .project(target: "Framework2", path: "../Framework2"),
+                                 ])
+])
+```
+
+### Project
+
+A `Project.swift` should initialize a variable of type `Project.swift`. It can take any name, although we recommend to stick to `project`. A project accepts the following attributes:
+
+- **Name:** Name of the project. It's used to determine the name of the generated Xcode project.
+- **Targets:** It contains the list of targets that belong to the project. Read more [about targets](#target)
+- **Settings (optional):** Read more about [settings](#settings)
+
+> **Note:** All the relative paths in the project manifest are relative to the folder that contains the manifest file.
+
+### Target
+
+Each target in the list of project targets can be initialized with the following attributes:
+
+- **Name:** The name of the target. The Xcode project target and the derivated product take the same name.
+- **Platform:** The platform the target product is build for. The following products are supported:
+  - **.app:** An application
+  - **.staticLibrary:** A static library.
```

Comment:
do we want to call out `.staticFramework` ?

if we do, I'd add a note:

- **.staticFramework:** A static framework. This is a regular framework product however is configured to be staticky linked.